src/app/components/shell/header/header.hbs

```diff
@@ -1,41 +1,56 @@
-0.5 0 -0.4 0-0.8-0.1-1.1-0.3 -0.3-0.2-0.6-0.4-0.8-0.7l-1.3 1.3c0.4 0.5 0.9 0.8 1.4 1 0.5 0.2 1.1 0.3 1.7 0.3 0.4 0 0.8 0 1.2-0.1s0.8-0.2 1.1-0.4c0.3-0.2 0.6-0.5 0.8-0.8 0.2-0.3 0.3-0.7 0.3-1.2 0-0.5-0.1-0.9-0.3-1.2C99.3 15.9 99 15.7 98.7 15.5z" style="fill:#5E6062;" /><path class="st5" d="M104.1 9.5h-2v2.3h-1.7v1.7h1.7v3.7c0 0.5 0 0.9 0.1 1.2 0.1 0.4 0.2 0.7 0.4 0.9 0.2 0.2 0.5 0.4 0.8 0.5 0.3 0.1 0.8 0.2 1.3 0.2 0.2 0 0.5 0 0.8-0.1 0.3 0 0.6-0.1 0.8-0.2V18c-0.1 0.1-0.3 0.2-0.5 0.2 -0.2 0-0.4 0.1-0.6 0.1 -0.4 0-0.7-0.1-0.9-0.4 -0.2-0.2-0.2-0.6-0.2-0.9v-3.5h2.2v-1.7h-2.2V9.5z" style="fill:#5E6062;" /><path class="st5" d="M114.6 13.8c-0.1-0.3-0.2-0.5-0.3-0.8 -0.1-0.3-0.3-0.5-0.6-0.7 -0.3-0.2-0.6-0.4-1-0.5 -0.4-0.1-0.9-0.2-1.4-0.2 -0.6 0-1.3 0.1-1.9 0.3 -0.6 0.2-1.1 0.5-1.6 1l1.1 1.1c0.3-0.3 0.6-0.5 1-0.6 0.4-0.2 0.8-0.2 1.2-0.2 0.5 0 1 0.1 1.3 0.4 0.3 0.2 0.5 0.6 0.5 1.1v0.2h-0.5c-0.3 0-0.7 0-1.1 0 -0.4 0-0.8 0.1-1.1 0.1 -0.4 0.1-0.7 0.2-1.1 0.3 -0.3 0.1-0.7 0.3-0.9 0.5 -0.3 0.2-0.5 0.5-0.6 0.8 -0.2 0.3-0.2 0.7-0.2 1.1 0 0.4 0.1 0.7 0.3 1.1s0.4 0.6 0.7 0.7c0.3 0.2 0.6 0.4 0.9 0.5 0.4 0.1 0.7 0.2 1.1 0.2 0.5 0 1-0.1 1.5-0.3 0.4-0.2 0.8-0.5 1.1-1h0v1.1h1.8v-4.6c0-0.2 0-0.4 0-0.7C114.7 14.3 114.7 14 114.6 13.8zM112.8 16.7c0 0.6-0.2 1-0.5 1.3 -0.3 0.3-0.8 0.5-1.5 0.5 -0.2 0-0.3 0-0.5-0.1 -0.2 0-0.3-0.1-0.5-0.2 -0.1-0.1-0.2-0.2-0.3-0.3 -0.1-0.1-0.1-0.3-0.1-0.5 0-0.3 0.1-0.5 0.3-0.7 0.2-0.2 0.5-0.3 0.8-0.4 0.3-0.1 0.6-0.1 1-0.2 0.3 0 0.7 0 0.9 0h0.4V16.7z" style="fill:#5E6062;" /><polygon class="st5" points="124.3 11.8 122 11.8 120.3 14.2 118.7 11.8 116.2 11.8 119 15.5 115.8 19.8 118.2 19.8 120.3 16.9 122.4 19.8 124.8 19.8 121.6 15.5 " style="fill:#5E6062;" /><polygon class="st5" points="129.9 11.8 129.2 13.5 128.5 11.8 128 11.8 128 14 128.3 14 128.3 12.1 128.3 12.1 129.1 14 129.3 14 130.1 12.1 130.1 12.1 130.1 14 130.4 14 130.4 11.8 " style="fill:#5E6062;" /><polygon class="st5" points="125.7 12.1 126.4 12.1 126.4 14 126.8 14 126.8 12.1 127.5 12.1 127.5 11.8 125.7 11.8 " style="fill:#5E6062;" /><path id="green" class="st11" d="M47.5 6.5c0 0.8-0.8 1.5-1.6 1.5L9.6 6.3c-0.9 0-1.5-0.8-1.5-1.6l0.2-3.2C8.3 0.6 9.1 0 9.9 0l36.2 1.7c0.9 0 1.5 0.8 1.5 1.6L47.5 6.5z" style="fill:#77AF42;" /><path id="grey" class="st5" d="M35.6 18.2c0 0.5-0.4 0.9-0.9 0.9H5.2c-0.5 0-0.9-0.4-0.9-0.9v-4.6c0-0.5 0.4-0.9 0.9-0.9h29.5c0.5 0 0.9 0.4 0.9 0.9V18.2z" style="fill:#5E6062;" /><path class="st12" d="M44.6 23.1c0 0.5-0.4 0.9-0.9 1l-33.2 1c-0.5 0-0.9-0.4-0.9-0.9l-0.1-2.7c0-0.5 0.4-0.9 0.9-1l33.2-1c0.5 0 0.9 0.4 0.9 0.9L44.6 23.1z" style="fill:#F4D019;" /><path id="blue" class="st36" d="M40.2 28.7c0 0.5-0.4 0.9-0.9 0.9H7.6c-0.5 0-0.9-0.4-0.9-0.9v-1.8c0-0.5 0.4-0.9 0.9-0.9h31.7c0.5 0 0.9 0.4 0.9 0.9V28.7z" style="fill:#00246A;" /><path class="st8" d="M41 10.5c0 0.4-0.6 0.7-1.3 0.7H1.3c-0.7 0-1.3-0.3-1.3-0.7V9c0-0.4 0.6-0.7 1.3-0.7h38.5C40.4 8.3 41 8.6 41 9V10.5z" style="fill:#F47641;" /></g></g><g id="nav"/><g id="Mobile"/><g id="guides" class="st45" style="display:none;" /></svg>
```

Comment:
Correct, I will replace the logo with a cleaner and smaller SVG once it's merged. Also, I have the animations ready in another branch that we can add later.